src/coreComponents/physicsSolvers/multiphysics/benchmarks/DeviatedPoroElasticWellbore_Injection.xml

```diff
@@ -5,6 +5,7 @@
   <Solvers
```

Comment:
Please prepare a coarse version of this example for integrated test.

Reply:
Done